x-pack/test/api_integration/apis/endpoint/alerts.ts

```diff
@@ -66,96 +76,182 @@ export default function({ getService }: FtrProviderContext) {
       });
 
       it('alerts api should return 400 when paging properties are below boundaries.', async () => {
+        const args: AlertListRequestQuery = {
+          page_size: 0,
+        };
         const { body } = await supertest
-          .get('/api/endpoint/alerts?page_size=0')
+          .get(buildUrl(null, args))
           .set('kbn-xsrf', 'xxx')
           .expect(400);
         expect(body.message).to.contain('Value is [0] but it must be equal to or greater than [1]');
       });
 
       it('alerts api should return links to the next and previous pages using cursor-based pagination', async () => {
+        const args: AlertListRequestQuery = {
+          page_index: 0,
+        };
         const { body } = await supertest
-          .get('/api/endpoint/alerts?page_index=0')
+          .get(buildUrl(null, args))
           .set('kbn-xsrf', 'xxx')
           .expect(200);
         expect(body.alerts.length).to.eql(10);
         const lastTimestampFirstPage = body.alerts[9]['@timestamp'];
         const lastEventIdFirstPage = body.alerts[9].event.id;
-        expect(body.next).to.eql(
-          `/api/endpoint/alerts?${nextPrevPrefix}&after=${lastTimestampFirstPage}&after=${lastEventIdFirstPage}`
+        expect(body.next).to.contain(
+          `after=${lastTimestampFirstPage}&after=${lastEventIdFirstPage}`
         );
       });
 
       it('alerts api should return data using `next` link', async () => {
+        const args: AlertListRequestQuery = {
+          after: ['1542789412000', 'c710bf2d-8686-4038-a2a1-43bdecc06b2a'],
+        };
         const { body } = await supertest
-          .get(
-            `/api/endpoint/alerts?${nextPrevPrefix}&after=1542789412000&after=c710bf2d-8686-4038-a2a1-43bdecc06b2a`
-          )
+          .get(buildUrl(null, args))
```

Comment:
as much as it isn't savvy, i rather prefer the straight forward all-on-one-line data for GET requests like we had, at least until they get much larger that is.